packages/os/dbus-1-system.toml

```
@@ -0,0 +1,40 @@
+[user.root]
+rules = [
+    # Allow activator interface
+    { send_destination = "org.freedesktop.DBus", send_interface = "org.freedesktop.systemd1.Activator", allow = true },
+    # Allow monitoring
+    { send_destination = "org.freedesktop.DBus", send_interface = "org.freedesktop.DBus.Monitoring", allow = true },
+    # Allow stats interface
+    { send_destination = "org.freedesktop.DBus", send_interface = "org.freedesktop.DBus.Debug.Stats", allow = true }
+]
+
+[default]
+rules = [
+    # Allow all users to connect
+    { user = "*", allow = true },
+    # Deny owning names
+    { own = "*", allow = false },
+    { send_type = "method-call", allow = false },
+    # Allow signals
+    { send_type = "signal", allow = true },
+    # Allow replies
+    # This is a useless rule, it is dropped from the rules at runtime
+    # { send_type = "method_return", send_requested_reply = true, allow = true },
+    # This is a uselss rule, it is dropped from the rules at runtime
+    # { send_type = "error", send_requested_reply = true, allow = true },
+    # Allow all receives
+    { receive_type = "method-call", allow = true },
+    # This is a userless rule, it is dropped from the rules at runtime
```

Is this supposed to be:
```suggestion
    # This is a useless rule, it is dropped from the rules at runtime
```